androidsvg/src/main/java/com/caverock/androidsvg/SVG.java

```diff
@@ -1683,32 +1683,45 @@ public String getId() {
 
    // One of the element types that can cause graphics to be drawn onto the target canvas.
    // Specifically: 'circle', 'ellipse', 'image', 'line', 'path', 'polygon', 'polyline', 'rect', 'text' and 'use'.
-   static abstract class GraphicsElement extends SvgConditionalElement implements HasTransform
+   public static abstract class GraphicsElement extends SvgConditionalElement implements HasTransform
    {
       Matrix  transform;
+      RectF rectF;
 
       @Override
       public void setTransform(Matrix transform) { this.transform = transform; }
 
       public abstract boolean contains(Point point);
+      public abstract float[] getXYPointsArray();
+
+
+      public RectF initRectFWithOffset(float offset) {

```

Comment:
Also is this the best place to put this method? More classes like `Circle` also subclass `GraphicsElement` -- does an `initRectFWithOffset` method make sense so high up in the hierarchy?